packages/features/insights/hooks/useInsightsColumns.tsx

```diff
@@ -68,6 +68,32 @@ export const useInsightsColumns = ({
           return dataTableFilter(cellValue, filterValue);
         },
       }),
+      columnHelper.accessor("bookingUid", {
```

Comment:
Are the accessor keys supposed to be duplicate for both `bookingUid` and `bookingLink`? @eunjae-lee

Reply:
actually `id: "bookingUid"` is what matters, and the accessor decides what we get via `info.getValue()`.